Kickstarter-iOS/Features/PledgeView/Controllers/NoShippingPledgeViewController.swift

```diff
@@ -645,7 +697,7 @@ private func applyRootStackViewStyle(_ stackView: UIStackView) {
 
 private func applyRootInsetStackViewStyle(_ stackView: UIStackView) {
   stackView.axis = NSLayoutConstraint.Axis.vertical
-  stackView.spacing = Styles.grid(4)
+  stackView.spacing = Styles.grid(3)
```

Comment:
Updated to `3` to be more consistent with Figma design

![image](https://github.com/user-attachments/assets/c0587984-5fcf-426c-a16d-419b6f16e58f)